docs/recipes/babelrc.md

```diff
@@ -30,6 +30,21 @@ You can override the default Babel configuration AVA uses for test transpilation
 }
 ```
 
+## Provide ES2015 Polyfills to your tests
+
+You might run into issues when relying on ES2015-features your current environment do not support. Since Ava isn't providing any polyfills per default you might want to enable them by adding `babel-polyfill` manually to the `require` section:
```

Comment:
@novemberborn I rewrite the description. I think it reads better now (even though my mother-tongue is German, so don't expect too much) but i have still a note on it:

- The documentation should probably elaborate on the distinction between the execution environment of tests and the execution environment of the tested code. The current wording may leave that as a question to the beginner.